src/layouts/Base.astro

```diff
@@ -46,6 +50,11 @@ const jsonLDSchema = JSON.stringify(
 			href="/fonts/source-sans-pro-latin-600.woff2"
 			crossorigin
 		/>
+		{
+			donationsEnabled && (
+				<link rel="dns-prefetch" href="https://js.stripe.com/v3" />
```

Comment:
🔍 If donations are enabled (either we are in prod or the local environment has Netlify env variables) we do the dns resolution for the Stripe API ahead of time

Reply:
`:smart-move:`